src/convertToObject.js

```diff
@@ -6,7 +6,16 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  return sourceString
+  .split(';')
+  .reduce((res, style) => {
```

Comment:
The variable name 'res' is not very descriptive. According to the checklist, avoid using generic names like 'result'. Consider using a more descriptive name that indicates its purpose, such as 'styleObject'.